compiler/natives/src/crypto/sha256/sha256.go

```diff
@@ -0,0 +1,67 @@
+// +build js
+
+package sha256
+
+import (
+	"errors"
+
+	"github.com/gopherjs/gopherjs/js"
+)
+
+var ErrWebCryptoFailed = errors.New("crypto: Web Crypto API call failed")
+
+// Get the SubtleCrypto interface
+// https://www.w3.org/TR/WebCryptoAPI/
+func getSubtle() *js.Object {
+	crypto := js.Global.Get("crypto")
+	if crypto == js.Undefined {
+		crypto = js.Global.Get("msCrypto") // for IE11
+		if crypto == js.Undefined {
+			return nil
+		}
+	}
+	subtle := crypto.Get("subtle")
+	if subtle == js.Undefined {
+		subtle = crypto.Get("webkitSubtle") // for Safari
+		if subtle == js.Undefined {
+			return nil
+		}
+	}
+	return subtle
+}
```

Comment:
Would it make more sense to execute this code once, at package init time, and save results.

Instead of calling it once for every `crypto.Sum256` call?

Similar to how `net/http` package does it for figuring out the transport to use.